casepro/statistics/migrations/0008_dailyminutetotalcount.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('statistics', '0007_populate_label_totals'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DailyMinuteTotalCount',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('item_type', models.CharField(help_text='The thing being counted', max_length=1)),
+                ('scope', models.CharField(help_text='The scope in which it is being counted', max_length=32)),
+                ('count', models.IntegerField()),
+                ('minutes', models.IntegerField()),
```

Comment:
We could maybe switch it to seconds? I just would have thought that responding/closing cases would be more on the scale of minutes than seconds. We also need to be careful about maxing out the `IntegerField`, but I don't think a 60x factor will make a big difference with that (we definitely don't want milliseconds though, that would be a 60000x factor)